### PR TITLE
Reputation and voting power added

### PR DESCRIPTION
**Pull request by jayserdny**
Here are the changes made in your project:

- Reputation is now shown next to the username
- Voting power is added in a new item below the username

screen shot 2018-02-05 at 9 32 22 pm

Note: the text is set to white just on my end. In your project it is black.

### CODE REVIEW

File: steembar.py [hunk 1]
```diff
@@ -36,11 +38,29 @@ def steemit_api(steemit_name):
     url = '{}get_state?path=@{}'.format(API, steemit_name)
     return fetch(url)
 
-def get_vp(steemit_name):
+
+def get_vp_rp(steemit_name):
```
Comment (jayserdny) [inline steembar.py]: Here is my function added to get the voting power and the reputation of the user

File: steembar.py [hunk 2]
```diff
@@ -14,8 +14,10 @@
 # Steem Api Exp : https://api.steemjs.com/get_state?path=@tolgahanuzun
 
 import requests
-import os=
+import os
 from math import ceil
```
Comment (jayserdny) [inline steembar.py]: This module is required to cell up the voting power

File: steembar.py [hunk 3]
```diff
@@ -14,8 +14,10 @@
...
+from math import log
```
Comment (jayserdny) [inline steembar.py]: This module is required to format the reputation of the user using log base e

File: steembar.py [hunk 4]
```diff
@@ -14,8 +14,10 @@
...
+from math import isnan
```
Comment (jayserdny) [inline steembar.py]: This one is required to confirm if reputation isNaN

**Comment by tolgahanuzun** [comment]
I liked these updates, thank you.

**Comment by jayserdny** [comment]
You are most welcome.

**Pull request merged**